tests/cmaes_test.cpp

```diff
@@ -23,7 +23,11 @@ using namespace ens::test;
  */
 TEST_CASE("CMAESLogisticRegressionTest", "[CMAESTest]")
 {
-  CMAES<> cmaes(0, -1, 1, 32, 200, 1e-3);
+  arma::mat lb(1, 1, arma::fill::value(-1));
+  arma::mat ub(1, 1, arma::fill::value(1));
+  BoundaryBoxConstraint b(lb, ub);
+
+  CMAES cmaes(0, b, 32, 200, 1e-3);
```

Comment:
A test for `EmptyTransformation` would be a good idea too. :+1:

Reply:
Alright!